ios/FluentUI.Demo/FluentUI.Demo/ReadmeViewController.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import FluentUI
+
+class ReadmeViewController: UIViewController {
+
+    private struct Constants {
```

Comment:
shouldn't private be all at the end based on our swift guidelines?

Reply:
You're right, I'll go over the guidelines again thanks!